src/content/plugins/internal-plugins.md

```diff
@@ -78,16 +71,6 @@ Prefetches `request` and dependencies to enable a more parallel compilation. It
 
 ## output
 
-### FunctionModulePlugin
```

Comment:
Removed in https://github.com/webpack/webpack/commit/69a545c4444d881c87e6462ea6da707f7793fc93